x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -437,6 +476,18 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               </EuiFormRow>
             </Fragment>
           )}
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.modelMemoryLimitLabel', {
+              defaultMessage: 'Model memory limit',
+            })}
+          >
+            <EuiFieldText
+              disabled={isJobCreated}
+              value={modelMemoryLimit}
+              onChange={e => setFormState({ modelMemoryLimit: e.target.value })}
+              isInvalid={modelMemoryLimit === ''}
```

Comment:
I assume the back end only accepts byte units? In which case, this should be validated, like is done in the anomaly detection job wizards:

![image](https://user-images.githubusercontent.com/7405507/68948911-dbb60880-07b0-11ea-8139-23e7c8df2d2b.png)

Reply:
Added validation and help text here: f8c50cfe758da1782208983286bf40e4fc06af1f